spec/models/work_spec.rb

```diff
@@ -649,21 +725,26 @@
 
       context "a blob already exists for one of the files" do
         let(:work1) do
-          work = FactoryBot.create(:draft_work)
-          blob = work.send(:s3_file_to_blob, file1)
-          blob.save
-          work.save
-          work
+          FactoryBot.create(:draft_work)
```

Comment:
Is this really testing the same thing?  The original test attached a second blob.  This one now only attaches one